python/in_toto_attestation/v1/statement.py

```diff
@@ -0,0 +1,45 @@
+#  Wrapper class for in-toto attestation Statement protos.
+
+import in_toto_attestation.v1.statement_pb2 as spb
+from in_toto_attestation.v1.resource_descriptor import ResourceDescriptor
+
+STATEMENT_TYPE_URI = 'https://in-toto.io/Statement/v1'
+
+class Statement:
+    def __init__(self, subjects=[], predicate_type='', predicate={}):
+        self.pb = spb.Statement()
+        self.pb.type = STATEMENT_TYPE_URI
+        self.pb.subject.extend(subjects)
+        self.pb.predicateType = predicate_type
+        self.pb.predicate.update(predicate)
+
+    @staticmethod
+    def copy_from_pb(proto):
+        stmt = Statement()
+        stmt.pb.CopyFrom(proto)
+        return stmt
+
+    def validate(self):    
+        if self.pb.type != STATEMENT_TYPE_URI:
+            raise ValueError('Wrong statement type')
+
+        if len(self.pb.subject) == 0:
+            raise ValueError('At least one subject required')
+
+        # check all resource descriptors in the subject
+        subject = self.pb.subject
+        for i, rdpb in enumerate(subject):
```

Comment:
As we never use `i`, why not:
```suggestion
        for rdpb in subject:
```

Reply:
🤦 sorry, I was sure I'd carefully looked for a use before adding the comment. Not carefully enough!